src/d3a/d3a_core/sim_results/plotly_graph.py

```diff
@@ -0,0 +1,584 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of D3A.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+import logging
+import plotly as py
+import plotly.graph_objs as go
+import pendulum
+from sortedcontainers import SortedDict
+
+from d3a.constants import TIME_ZONE
+from d3a.models.strategy.storage import StorageStrategy
+from d3a.models.strategy.load_hours import LoadHoursStrategy
+from d3a.models.strategy.pv import PVStrategy
+from d3a import limit_float_precision
+
+_log = logging.getLogger(__name__)
+
+ENERGY_BUYER_SIGN_PLOTS = 1
+ENERGY_SELLER_SIGN_PLOTS = -1 * ENERGY_BUYER_SIGN_PLOTS
+
+alternative_pricing_subdirs = {
+    0: "d3a_pricing",
+    1: "no_scheme_pricing",
+    2: "feed_in_tariff_pricing",
+    3: "net_metering_pricing"
+}
+
+EXPORT_DEVICE_VARIABLES = ["trade_energy_kWh", "pv_production_kWh", "trade_price_eur",
+                           "soc_history_%", "load_profile_kWh"]
+
+green = 'rgba(20,150,20, alpha)'
+purple = 'rgb(156, 110, 177, alpha)'
+blue = 'rgba(0,0,200,alpha)'
```

Comment:
Can you tell me the reason to have rgba for 'green' and 'blue, whereas rgb for purple?

Reply:
This is a typo. Will fix it. Thanks